README.md

```diff
@@ -103,8 +103,14 @@ Import all my code to Github
 
 ## Data Cleaning 
 
-Import all my code to Github that I have cleaned
-Explain what I have done and why?
+##what I have cleaned and why?
+
+When I began coding/visualing my chosen dataset **"covid19_open_data"**, I realised that there must be negative values recorded due to how the graphs where presented👇. 
+
+![Screenshot from Deepnote](Negative_Values_Graph.png)
+
+This negativly affected my project as the data I was collecting couldn't have values lower then zero. This was due to the fact that the data recorded eg **'school_closing'**, **'workplace_closing'** and **'public_transport_closing'** had to have a value of 1,2 or 3 and **'new_confirmed'** and **'new_deceased'** had to have a value of 0. Finally, **'country_name'** had to be an name rather then number and **'date'** had to be in the format 2020-11-13. Therfore, the data recorded would result in all my code/visualisation being inaccurate. 
```

Comment:
May I ask you to change the wording here: you WERE not collecting any data. Someone else did.

Reply:
Hi Katia, I can't edit it till I merge it, but would this be ok? "This negatively affected my project as the data that had been collected couldn't have values lower than zero."

Reply:
Hi Katia, I have edited the data cleaning section with the above and created a new pull request for yourself to approve :)